src/libmongoc/tests/test-mongoc-dns.c

```diff
@@ -633,8 +633,8 @@ _prose_loadbalanced_ping (mongoc_client_t *client)
 }
 
 /*
-   Implements the following prose test described in the SRV polling test README:
-   Test that SRV polling is not done for load balalanced clusters. Connect to
+   Implements prose test 9 as described in the SRV polling test README:
+   Test that SRV polling is not done for load balanced clusters. Connect to
```

Comment:
Added a minor typo fix to the set of changes which I had forgotten to include prior.

Reply:
Good eye!